### ISSUE TITLE
Add "Looking for Game" functionality

### ISSUE THREAD

**Issue by db0** [octgn/OCTGN#109]
[Repost from Skylabs forum](https://www.skylabsonline.com/forum/discussion/107/add-looking-for-game-functionality#Item_4)

I play and know how to play a lot of CCGs, but almost all of them are very obscure, to the point that I'm lucky if there's anyone even logged in who plays one of them. However, at any given time, I'm open to playing any of them given the opportunity. Unfortunately the way that lobbies work, I can only inform available players that I'm available for only one specific game. Thus if I make a lobby for Doomtown, people who have only 7th Sea will not know that I would be able to play 7th Sea as well given the chance. Furthermore, [given the way the non-afk-lobbies work](https://www.skylabsonline.com/forum/discussion/99/using-the-lobby-while-hosting), I can't even leave my lobby active and go see if someone else is hosting something else I'd like to play.

What I'd like to suggest is either of two options. Either a way to host a lobby for more than one game and then select them just before starting the game, OR a way to mark oneself with what games you are available, and then a tab to show how many players are online and available for each game and even a way to PM those players to setup a game.

**Comment by brine** [comment]
most of this is based on old infrastructure we no longer have so this will be closed.  I think the current goal is to have invite links via discord, which would be a separate ticket at this point
